dashboard/app/controllers/openai_evaluate_controller.rb

```diff
@@ -39,6 +39,12 @@ def evaluate
       aiReasoning: "The response contains profanity and could not be evaluated.",
     }
 
+    pii_detected_response = {
+      aiEvaluation: "PII detected",
+      evaluationCriteria: "Does the student work contain personally identifying information?",
+      aiReasoning: "The response contains PII and could not be evaluated.",
```

Comment:
If we decide to show the response in the table, which I think we will, I wonder if we should spell out PII here so teachers know what it means.